pages/etablissement/[slug].tsx

```diff
@@ -74,9 +87,17 @@ export const getServerSideProps: GetServerSideProps = postServerSideProps(
     const etablissementWithUniteLegale =
       await getEtablissementWithUniteLegaleFromSlug(slug, isBot);
 
+    const servicePublic =
+      isBot || !isServicePublic(etablissementWithUniteLegale.uniteLegale)
+        ? null
+        : await getServicePublicByEtablissement(
+            etablissementWithUniteLegale.etablissement
+          );
```

Comment:
You can move the isbot and the isServicePublic condition into the model `getServicePublicByEtablissement` as we do for `getEtablissementWithUniteLegaleFromSlug`. it will avoid having the condition written twice, once in `pages/etablissement/[slug].tsx`and once in `pages/entreprise/[slug].tsx`